collector/serverstats.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 Ben Kochie
```

Comment:
Nit, new files get the current year.
```suggestion
// Copyright 2024 Ben Kochie
```